build/make-kernel.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ARCH=`uname -m`
+ARCHPATH=arch/$ARCH
+
+cp $ARCHPATH/kernel_config ./kernel_config
+cp Dockerfile.$ARCH Dockerfile
+
+# Till the ppc64le gets hyperctl ported it has to use 
+# stardard docker to generate kernel binary
+if [ $ARCH == "ppc64le" ] 
+then 
+	docker build -t hyperstart-dev-$ARCH:latest -f Dockerfile . 
+	docker run --rm hyperstart-dev-$ARCH:latest cat /root/build/result/kernel >kernel.new && mv -f kernel.new $ARCHPATH/kernel 
```

Comment:
If using hyper to build the kernel is workable after bootstrapped on ppc64le, please use hyper. Otherwise, could you please report the issue about it?

docker is allowed to be used anytime for the same purpose, especially when bootstrapping or when some problem occurring. But merging the kernel-building code using docker is not encouraged.

Reply:
@laijs Right now hyper is not a workable solution because hyper has not been ported to ppc64le yet. I can understand your concerns about having a code that explicitly uses docker, but on architectures where hyper is not yet available there is no other choice. 

Would it make sense if I push a README instead of the code change? That way our users won't be left to guessing on how to build a kernel. Of course, this might change if hyper ported to ppc64le, but till then I believe there has to be a way to communicate with the users to use docker. 

What do you say?

Reply:
Ok, for now, I will update this PR by dropping the kernel build code. Thanks for your inputs.

Reply:
Thanks. And if there is any problem for the hyper-kernel-build on ppc64le after this pr merged, don't hesitate to report it.